src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useRef, useState } from 'react';
+import { Person } from '../types/Person';
```

Comment:
Verify that the `Person` type includes the `slug` and `sex` properties, as these are used in the component for rendering and styling purposes.